azext_edge/edge/params.py

```diff
@@ -540,6 +540,15 @@ def load_iotops_arguments(self, _):
                 arg_group="Trust",
             )
 
+    with self.argument_context("iot ops upgrade") as context:
+        # Schema Registry
+        context.argument(
+            "schema_registry_resource_id",
+            options_list=["--sr-resource-id"],
+            help="The schema registry resource Id to use with IoT Operations. Required if the schema registry "
+            "resource Id is no longer found within IoT Operations.",
+        )
```

Comment:
Would it make sense to just change the previous grouping to `for cmd_space in ["iot ops init", "iot ops create", "iot ops upgrade"]:` so we only need the schema registry resource ID defined once?

Or are there other parameters used in the method header that will get picked up by the args in init and create?

Reply:
the help is different so I will keep it out. Also makes it easier to remove